src/Acquisition/__init__.py

```diff
@@ -21,8 +21,11 @@
 IS_PURE = 'PURE_PYTHON' in os.environ
 
 
-class Acquired(object):
-    "Marker for explicit acquisition"
+# Bring this in line with the "C" implementation
+# Zope requires this to be an `str`
+##class Acquired(object):
+##    "Marker for explicit acquisition"
```

Comment:
I think this comment is not needed as it describes something that's already expected. (C implementation behaves the same like the Python one.)